doc/source/tutorials/basic_usage.rst

```diff
@@ -0,0 +1,89 @@
+
+Other force fields are available to convert your protein to. To view them you
+can use ``martinize2 -list-ff``.
+
+Minimal physical reality usage
+==============================
+
+Our knowledge of proteins and Martini tells us that we need to add some more
+information to the topology to account for secondary structure.
+
+Let martinize2 deal with secondary structure for you
+----------------------------------------------------
+
+Martinize2 can deal with secondary structure intelligently using dssp in one of two ways:
+
+1) Use a dssp executable installed on your machine
+2) Use the dssp implementation in `mdtraj <https://mdtraj.org/1.9.4/api/generated/mdtraj.compute_dssp.html>`_
+
+To explain these more:
+
+1) If you have dssp installed locally, note that martinize2 is only validated for particular versions of dssp.
+   If a non-validated version is used, a warning will be raised and nothing is written.
+   If you know what you're doing and are confident with what's been produced, you can override the warning
+   with the ``-maxwarn`` flag. Otherwise, dssp can be used using the ``-dssp`` flag in martinize.
+
+   Where you have a local installation, replace ``/path/to/dssp`` in the following command with the
+   location of your installation:
+
+   ``martinize2 -f protein.pdb -o topol.top -x cg_protein.pdb -ff martini3001 -dssp /path/to/dssp``
+
+2) If you have `mdtraj <https://mdtraj.org/1.9.4/api/generated/mdtraj.compute_dssp.html>`_ installed in
+   your python environment, the ``-dssp`` flag can be left blank, and martinize2 will attempt to use
+   dssp from there:
+
+   ``martinize2 -f protein.pdb -o topol.top -x cg_protein.pdb -ff martini3001 -dssp``
+
+
+User knows best
+---------------
+
+If you already know the secondary structure of your protein and don't want to worry about
+dssp assigning it correctly, the ``-ss`` flag can be used instead.
+
+The ``-ss`` flag must be one of either:
+
+1)   The same length as the number of residues in your protein, with a dssp code for each residue
+2)   A single letter (eg. '``H``'), which will apply the same secondary structure parameters to your entire protein.
+
+For example:
+
+``martinize2 -f protein.pdb -o topol.top -x cg_protein.pdb -ss HHHHHHHHHH``
+
+will read the ``HHHHHHHHHH`` dssp formatted string, indicating that there are exactly 10 residues in the
+input pdb which should all be treated as part of a helix. If the string provided does not contain the same
+number of residues as the input file, an error will be raised.
+
+Alternatively in this case, as we know everything should be a helix, the same result can be achieved through
+using a single letter as described above:
+
+``martinize2 -f protein.pdb -o topol.top -x cg_protein.pdb -ss H``
+
+If instead we needed to assert that only the first five residues are in a helix, and the final five are coiled,
+we must we the full length string again:
+
+``martinize2 -f protein.pdb -o topol.top -x cg_protein.pdb -ss HHHHHCCCCC``
+
+
+
+
```

Comment:
-  think we would want to add some text about the -scfix , which is not default and for sure should be added with martini3
- bonus for one sentence on cysteine bridges 
- perhaps in the end here we may want to hint at EN and fine tuning of biases. Ultimately this is were people will start if they have no clue.
- Something about the citations it prints in the end as well? We could say that they are dynamically collected and therefore should all be cited. Not a major point but nice to know.

Reply:
see also #620